src/ocean/text/util/StringC.d

```diff
@@ -89,7 +89,9 @@ class StringC
     {
         if (str.length && !!str[$ - 1])
         {
+            enableStomping(str);
             str ~= StringC.Term;
```

Comment:
Hmm, I thought there was a consensus that in library functions one should call `enableStomping` before and after changing the length of the user-provided buffer, just because this reason (because the library can't assert on the fact that user did call `enableStomping` where necessary). I have no problems to remove this call, if that's not the case.